restangular/restangular.d.ts

```diff
@@ -124,7 +113,7 @@ declare module restangular {
     clone(): IElement;
     plain(): any;
     plain<T>(): T;
-    withHttpConfig(httpConfig: IRequestConfig): IElement;
+    withHttpConfig(httpConfig: ng.IRequestShortcutConfig): IElement;
```

Comment:
I know it has changed, but I wanted to keep file consistent (other angular references use "ng"). I can change it in whole file if needed :)